js/README.md

```diff
@@ -100,55 +100,163 @@ Subdirectories with additional utilities
 This section describes how to add new blocks to Music Blocks in order
 to add functionality.
 
-* You can add an individual block to a palette by modifying two files:
-`basicblocks.js` and `logo.js`.
+<!-- * You can add an individual block to a palette by modifying two files:
+`basicblocks.js` and `logo.js`. -->
+* To add a new block you first need to look through `js/blocks/` and determine if a new file is needed or not. If required then go through the section down below on how to declare new files for blocks requiring so.
 
-* If the block you are adding needs to expand into a stack of blocks,
-you may also need to modify `macro.js`.
+## How to define new files for blocks
 
-* If you want to add a new palette with multiple blocks for a specific
+Note: You may skip this section if the block you're trying to add doesn't require a new file.
```

Comment:
See explanation above about when you need a new file.

Reply:
Also corrected this one.